### PR TITLE
Add keymap for MOTHWING

### PR DESCRIPTION
**Pull request by tan-t**
## Description

added a keymap for "MOTHWING" keyboard.

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/19342
(reviewed, but not merged yet)

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/mothwing/mothwing.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "vendorId": "0x4D77",
+  "productId": "0x0001",
+  "matrix": {"rows": 8, "cols": 6},
+  "customKeycodes": [
+    {
+      "name": "Zenkaku/Hankaku",
+      "title": "Toggle Zenkaku Hankaku",
+      "shortName": "ZenHan"
+    }
+  ],
```
Comment (Xelus22) [inline v3/mothwing/mothwing.json:12]: I dont' see a custom keycode in your QMK firmware

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please remove V2 definition as that is only legacy support boards (`src/` folder)

File: v3/mothwing/mothwing.json (hunk 1 above)
Comment (tan-t) [reply v3/mothwing/mothwing.json:12]: oh thanks

**Comment by tan-t** [comment]
> Please remove V2 definition as that is only legacy support boards (src/ folder)
I see

**Comment by tan-t** [comment]
@Xelus22 
sorry for the late reply.  
I deleted legacy settings, and deleted `customKeycodes` (that was for testing, I forgot to delete it)

**Comment by tan-t** [comment]
@Xelus22 sorry to bother you, but can you have it merged by the end of July? thanks!

File: v3/mothwing/mothwing.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "MOTHWING",
+  "vendorId": "0x4D77",
+  "productId": "0x0001",
+  "matrix": {"rows": 8, "cols": 6},
+  "layouts": {
+    "labels": [],
```
Comment (Xelus22) [inline v3/mothwing/mothwing.json:7]: Please remove empty label

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/mothwing/mothwing.json (hunk 2 above)
Comment (tan-t) [reply v3/mothwing/mothwing.json:7]: sure!

Comment (tan-t) [reply v3/mothwing/mothwing.json]: I deleted it.

**Comment by tan-t** [comment]
@Xelus22 Hi! I think I resolved all the request for changes. could you check it? thanks!

**Comment by tan-t** [comment]
@Xelus22 I'm really sorry to bother you but can you merge it soon? my first copies will be arrived tommorrow or something.  
I'm really sorry but if you could merge it soon I will appreciate it. thank you

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**